src/utils/Util.ts

```diff
@@ -0,0 +1,21 @@
+import Message from "../Message";
+
+export default class Util {
+    /**
+     * Removes Dangerous pings such as @everyone and @here from the message.
+     * 
+     * @param message The message object
+     * @see {Message}
+     */
+    static removeMentions(message: Message): Message {
```

Comment:
We separate every methods to `utils/<utility_name>.ts`. Could you move this method to a file rather than make it as a static class method?

For example, `utils/removeMentions.ts`.